<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Transactions</class>
 <widget class="QDialog" name="Transactions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1162</width>
    <height>613</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="WelcomeLabel">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>461</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Welcome :  </string>
   </property>
  </widget>
  <widget class="QLabel" name="AccLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>120</y>
     <width>441</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Account number : </string>
   </property>
  </widget>
  <widget class="QLabel" name="BalLabel">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>160</y>
     <width>261</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Currrent balance  : </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>220</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Amount  : </string>
   </property>
  </widget>
  <widget class="QLineEdit" name="AmountField">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>220</y>
     <width>201</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="ClearBut">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>220</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DepositBut">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>280</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Deposit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="WithdrawBut">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>280</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Withdraw</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ShowBut">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>400</y>
     <width>121</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Show History:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="HideBut">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>400</y>
     <width>111</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Hide History</string>
   </property>
  </widget>
  <widget class="QPushButton" name="BackBut">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>450</y>
     <width>83</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Back</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>710</x>
     <y>110</y>
     <width>261</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="statusL">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>350</y>
     <width>151</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
